network-api/networkapi/buyersguide/utils.py

```diff
@@ -41,3 +52,42 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
 
     def describe(self):
         return "Update %s bases to %s" % (self.name, self.bases)
+
+
+def convert_pni_file_to_product_page_image(pni_product: Product, new_product_page: ProductPage):
+    """
+    Take an existing product and convert it's FileField image to a WagtailImage object.
+
+    pni_product must be an instance of a Product
+    new_product_page must be an instance of a ProductPage
+    """
+    # 1. Get the mimetype of the image.
+    mime = MimeTypes()
+    mime_type = mime.guess_type(pni_product.image.file.name)  # -> ('image/jpeg', None)
+    if mime_type:
+        mime_type = mime_type[0].split('/')[1].upper()
+    else:
+        # Default to a JPEG mimetype.
+        mime_type = 'JPEG'
```

Comment:
this is probably fine but we'll need to test this once we have all the code in place

Reply:
I agree. Ill be doing extensive testing when we start migrating snippets to Pages (locally), too.